### ISSUE TITLE
Extra save-file for dedicated servers

### ISSUE THREAD

**Issue by Nubbie** [OpenRCT2/OpenRCT2#4152]
The game is able to run in a dedicated mode via the command
```
openrct2 host <saved game path> --port 11753 --headless
```
But it has no way to load the latest autosave from a crashed game as it creates a file called 'autosave_%time%.sv6' with the time updated constantly (making it impossible as it doesn't have a static name)
I would like to have a checkbox that, when ticked, creates a extra save-file called 'DedicatedServer.sv6' 
In that way, the dedicated server command would be working as all you need to do is type;

```
openrct2 host "openrct2/saves/dedicatedserver.sv6" --port 11753 --headless
```

![sdfggsd](https://cloud.githubusercontent.com/assets/17282384/17071208/65b29348-5060-11e6-8171-23bfe845cf7c.png)

**TL;DR:** Extra copy of the latest autosave with static name

**Comment by janisozaur** [comment]
Pointless, just load using the newest autosave:
```
./openrct2 $(ls -t1 | head -n 1)
```
You can add some filters to check for `sv6` and `autosave` as well.

**Comment by Nubbie** [comment]
@janisozaur Just a bit too dumb to know the full command to do that :U

This is all I understand..
```
openrct2 host "C:\Users\x\Documents\Autosave.sv6" --port 11753 --silent-breakpad 
```
